bigbluebutton-tests/playwright/learningdashboard/learningdashboard.spec.js

```
@@ -21,8 +21,8 @@ test.describe('Learning Dashboard', { tag: '@ci' } , async () => {
     await learningDashboard.userTimeOnMeeting();
   });
 
-  test('Polls', { tag: '@flaky' }, async () => {
-    await learningDashboard.initUserPage(true, learningDashboard.modPage.context, { isRecording: true });
+  test('Polls', { tag: '@flaky' }, async ({}, testInfo) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)